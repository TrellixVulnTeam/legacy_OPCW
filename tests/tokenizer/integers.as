3242324
3462778959793846834
4352345
324673289732978
0b01101011010
0B10101
0x0FABCDE
0X320942
0O1123214
0o755
