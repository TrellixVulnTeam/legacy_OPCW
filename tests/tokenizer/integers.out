<decimal integer> '3242324'
<decimal integer> '3462778959793846834'
<decimal integer> '4352345'
<decimal integer> '324673289732978'
<binary integer> '01101011010'
<binary integer> '10101'
<hexadecimal integer> '0FABCDE'
<hexadecimal integer> '320942'
<octal integer> '1123214'
<octal integer> '755'
<end>
